File: CourseRegistrationSystem/src/CourseSection.java

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class CourseSection {

    @JsonProperty("term")

    private String term;
    @JsonProperty("semester")

    private int semester;
    @JsonProperty("instructor")

    private String instructor;
    @JsonProperty("enrollmentCapacity")

    private int enrollmentCapacity;
    //private List<Student> enrolledStudents;
    @JsonProperty("status")

    private String status;

    public CourseSection(String term, int semester, String instructor, int enrollmentCapacity,String status) {
        this.term=term;
        this.instructor = instructor;
        this.status = status;
        this.enrollmentCapacity = enrollmentCapacity;
        this.semester=semester;
        //this.enrolledStudents = enrolledStudents;
        
    }

    public CourseSection(){
    }
    
    //public List<Student> getEnrolledStudents(){
    //
    //}


    public String getTerm() {
        return term;
    }
    public int getSemester(){return semester;}

    public String getInstructor() {
        return instructor;
    }

    public int getEnrollmentCapacity() {
        return enrollmentCapacity;
    }

    public String getStatus() {
        return status;
    }
    public String toString(){
        StringBuilder sb = new StringBuilder();
        sb.append("CourseSection[term=").append(term)
          .append(", instructor=").append(instructor)
          .append(", status=").append(status)
          .append(", enrollmentCapacity=").append(enrollmentCapacity)
          //.append(", enrolledStudents=").append(enrolledStudents)
          .append("]");
        return sb.toString();
    }


}

File: iteration3/CourseSystem/CourseSection.py
class CourseSection:
    def __init__(self, term="", day="", hour="", semester=0, instructor="", enrollment_capacity=0, status=""):
        self.term = term
        self.day = day
        self.hour = hour
        self.semester = semester
        self.instructor = instructor
        self.enrollment_capacity = enrollment_capacity
        self.status = status

    def get_term(self):
        return self.term

    def get_day(self):
        return self.day

    def get_hour(self):
        return self.hour

    def get_semester(self):
        return self.semester

    def get_instructor(self):
        return self.instructor

    def get_enrollment_capacity(self):
        return self.enrollment_capacity

    def get_status(self):
        return self.status

    def __str__(self):
        return (f"CourseSection[term={self.term}, instructor={self.instructor}, status={self.status}, "
                f"enrollmentCapacity={self.enrollment_capacity}]")
